plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -196,10 +199,11 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "By default, the texture cache handles partial invalidations. Unfortunately it is very costly to compute CPU wise."
 				"\n\nThis hack replaces the partial invalidation with a complete deletion of the texture to reduce the CPU load.\n\nIt helps snowblind engine games.";
 		case IDC_LARGE_FB:
-			return "Allocate a large framebuffer to be compliant with GS memory (Prevents FMV flickering).\n"
-				"It increases GPU/memory requirements.\n\n"
-				"Note: It should be disabled for Armored Core, Destroy All Humans, and Gran Turismo.\n"
-				"It will amplify RAM/VRAM spikes. Other games might be affected as well.";
+			return "Reserves a larger framebuffer for textures to prevent FMV flickers.\n"
```

Comment:
I don't think this is for textures; isn't this for the actual framebuffer used to draw to screen? Maybe just remove "for textures" and that would be all it takes.

Reply:
Console changes stuff like 512x512 to 512x3012 if enabled. I'll remove the words though.